test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua

```diff
@@ -20,7 +20,7 @@
 --  Expected behavior:
 --  1. App has successfully registered.
 --  2. App is disconnected by SDL due to heartbeat timeout occurs.
-
+---------------------------------------------------------------------------------------------------
```

Comment:
@dboltovskyi Please create separate commits with style fixes and functional fixes. 
This lets reviewers concentrate on functional fixes.

Reply:
@aderiabin It will take sometime to do this. Do we really need this? Current PR was done in strict time schedule in order to have correct smoke tests.